pkg/network/ovn_kubernetes_test.go

```diff
@@ -189,7 +199,7 @@ func TestRenderedOVNKubernetesConfig(t *testing.T) {
 		disableGRO               bool
 		disableMultiNet          bool
 		enableMultiNetPolicies   bool
-		enableAdminNetPolicies   bool
+		featureGates             configv1.CustomFeatureGates
```

Comment:
I am thinking, maybe to make less changes in the future, we could replace this parameter with `enableFeatureGates []FeatureGateName` and then build a real config by iterating over all known feature gates and adding the ones that are listed to `Enabled` and others to `Disabled`. In that case you don't need to change tests that don't enable any feature gates, and only make changes when a new feature gate should be enabled. What do you think?

Reply:
Agreed, and that's a good suggestion. I've updated the structure, hope it looks cleaner now.